client/src/main/java/com/alibaba/nacos/client/naming/remote/gprc/redo/RedoScheduledTask.java

```diff
@@ -75,6 +76,12 @@ private void redoForInstance(InstanceRedoData redoData) throws NacosException {
                 if (isClientDisabled()) {
                     return;
                 }
+                if (redoData instanceof BatchInstanceRedoData) {
```

Comment:
abstract line from 79 to 85 into other method.

Reply:
> abstract line from 79 to 85 into other method.

i abstract it